tests/mocks/ec-focus-trap.mock.js

```diff
@@ -1,16 +0,0 @@
-jest.mock('../../src/directives/ec-focus-trap', () => {
```

Comment:
We need to create a new global mock that will work in EBO when using modals in the unit tests.